Multisig/UI/ClaimToken/ChooseDelegateIntroViewController.swift

```diff
@@ -38,8 +36,12 @@ class ChooseDelegateIntroViewController: UIViewController {
         navigationItem.rightBarButtonItem = UIBarButtonItem(customView: stepLabel)
         stepLabel.setStyle(.tertiary)
         stepLabel.text = "\(stepNumber) of \(maxSteps)"
+
         titleLabel.setStyle(GNOTextStyle.Updated.title)
+
```

Comment:
Do we need all of these empty lines

Reply:
they introduce vertical grouping of lines, so that lines that are related to changing similar things are in a similar group